src/main/java/org/mifos/pheevouchermanagementsystem/camel/routes/AccountLookupRoute.java

```diff
@@ -29,6 +30,7 @@ public void configure() throws Exception {
             String callbackUrl = exchange.getProperty(CALLBACK, String.class);
             exchange.getIn().setHeader(CALLBACK, callbackUrl);
             exchange.getIn().setHeader("X-Registering-Institution-ID", exchange.getProperty(REGISTERING_INSTITUTION_ID));
+            exchange.getIn().setHeader("isExternalLookup", exchange.getProperty(IS_EXTERNAL_LOOKUP));
```

Comment:
are we still using the header approach?

Reply:
No, there were some left over parts, which I have removed now.